db/migrate/20230719070614_create_users.rb

```diff
@@ -0,0 +1,14 @@
+# frozen_string_literal: true
+
+class CreateUsers < ActiveRecord::Migration[7.0]
+  def change
+    create_table :users do |t|
+      t.string :name
+      t.text :photo
+      t.text :bio
+
+      t.timestamps
+    end
+    add_index :users, 
+  end
+end
```

Comment:
- [ ] Good job so far 👍👍 However, While running migration, I am getting the following error  and I am unable to see your implementation in accordance with the previous reviewer's suggestion⚠👇 Kindly make sure to fix this 
- [ ] Also, all the columns as as stated in the [ERD diagram](https://github.com/microverseinc/curriculum-rails/blob/main/blog-app/images/blog_app_erd.png) are not being added 😓 kindly fix this as suggested earlier.

![image](https://github.com/KAYEMBAJONATHAN/Blog-app-/assets/106119105/25982204-6896-4b85-955f-a1d02b8d0b3a)